src/Products/ZCatalog/ZCatalog.py

```diff
@@ -410,14 +410,12 @@ def manage_clearIndex(self, ids=None,
 
     @security.protected(manage_zcatalog_entries)
     def reindexIndex(self, name, REQUEST, pghandler=None):
```

Comment:
I'd like to see a docstring added, as it is not entirely clear what name is, or should be. Obviously it can be a string or something else.